corehq/form_processor/tests/test_sharding.py

```diff
@@ -19,7 +20,8 @@ class ShardingTests(TestCase):
 
     @classmethod
     def setUpClass(cls):
-        cls.partion_config = PartitionConfig()
+        with trap_extra_setup(PartitionedDatabaseRequiredError):
```

Comment:
Why is this necessary? Does nose not respect the `@skipUnless` decorator on the class?